src/applications/health-care-supply-reordering/components/IntroductionPage.jsx

```diff
@@ -110,10 +114,105 @@ const IntroductionPage = ({ route }) => {
             prefillEnabled={route.formConfig.prefillEnabled}
             messages={route.formConfig.savedFormMessages}
             pageList={route.pageList}
-            startText={`Order ${supplyDescription}`}
+            startText="Start your hearing aid and CPAP supplies order"
             unauthStartText="Sign in to start your order"
             formConfig={route.formConfig}
           />
+          <h4 className="vads-u-font-size--h2">Frequently asked questions </h4>
```

Comment:
Recommend changing this to an <h2> to maintain proper heading hierarchy.
`<h4 className="vads-u-font-size--h3">Frequently asked questions </h4>`